docs/content/development/libraries.md

```diff
@@ -31,6 +31,7 @@ Some great folks have written their own libraries to interact with Druid
 #### Ruby
 
 * [madvertise/ruby-druid](https://github.com/madvertise/ruby-druid) - A ruby client for Druid
```

Comment:
not worth a pull request on it's own, but can you fix that url to point https://github.com/ruby-druid/ruby-druid ? Thanks